Documentation/network/servicemesh/gateway-api/grpc.rst

```diff
@@ -0,0 +1,161 @@
+routes requests to a gRPC service (i.e. using HTTP/2). In order for this example to
+work, ALPN support needs to be enabled with the Helm flag ``gatewayAPI.enableAlpn``
+set to true. This enables clients to request HTTP/2 through the TLS negotiation.
+
+.. literalinclude:: ../../../../examples/kubernetes/gateway/grpc-tls-termination.yaml
+     :language: yaml
+
+.. tabs::
+
+    .. group-tab:: Self-signed Certificate
+
+        For demonstration purposes we will use a TLS certificate signed by a made-up,
+        `self-signed <https://cert-manager.io/docs/faq/terminology/#what-does-self-signed-mean-is-my-ca-self-signed>`_
+        certificate authority (CA). One easy way to do this is with `mkcert <https://github.com/FiloSottile/mkcert>`_.
+        We want a certificate that will validate ``grpc-echo.cilium.rocks``, as these are the host names used in this example.
+
+        .. code-block:: shell-session
+
+            $ mkcert bookinfo.cilium.rocks hispter.cilium.rocks
+            Created a new local CA 💥
+            Note: the local CA is not installed in the system trust store.
+            Run "mkcert -install" for certificates to be trusted automatically ⚠
+
+            Created a new certificate valid for the following names 📜
+             - "grpc-echo.cilium.rocks"
+
+            The certificate is at "./grpc-echo.cilium.rocks.pem" and the key at "./grpc-echo.cilium.rocks-key.pem" ✅
+
+            It will expire on 28 September 2027 🗓
+
+        Create a Kubernetes secret with this demo key and certificate:
+
+        .. code-block:: shell-session
+
+            $ kubectl create secret tls grpc-certificate --key=grpc-echo.cilium.rocks-key.pem --cert=grpc-echo.cilium.rocks.pem
+
+    .. group-tab:: cert-manager
+
+        Let us install cert-manager:
+
+        .. code-block:: shell-session
+
+            $ helm repo add jetstack https://charts.jetstack.io
+            $ helm install cert-manager jetstack/cert-manager --version v1.16.2 \
+                --namespace cert-manager \
+                --set crds.enabled=true \
+                --create-namespace \
+                --set config.apiVersion="controller.config.cert-manager.io/v1alpha1" \
+                --set config.kind="ControllerConfiguration" \
+                --set config.enableGatewayAPI=true
+
+        Now, create a CA Issuer:
+
+        .. parsed-literal::
+
+            $ kubectl apply -f \ |SCM_WEB|\/examples/kubernetes/servicemesh/ca-issuer.yaml
+
+Deploy the Gateway and GRPCRoute
+================================
+
+This sets up a simple gRPC echo server and a Gateway to expose it.
+
+.. tabs::
+
+    .. group-tab:: Self-signed Certificate
+
+        We already created the certificate Secret in the previous step, so
+        we can just create the resources:
+
+        .. parsed-literal::
+
+            $ kubectl apply -f \ |SCM_WEB|\/examples/kubernetes/gateway/grpc-tls-termination.yaml
+
```

Comment:
```suggestion
        .. parsed-literal::

            $ kubectl apply -f \ |SCM_WEB|\/examples/kubernetes/gateway/grpc-tls-termination.yaml

        The self-signed certificate Secrets from the previous step will be used by this Gateway.
```

Reply:
^ The goal here is to avoid the use of "we", and also to match the same flow as the other tab - hence why I suggested an alternate sentence after the common step rather than before.